doc/manual/manual.tex

```diff
@@ -8286,6 +8289,12 @@ \subsubsection{The van Keken thermochemical composition benchmark}
 
 We can observe that when the two-layer viscosity model is added, there is only one apparent peak for each case. The first peaks of the 0.01 viscosity contrast tests appear earlier and are larger in magnitude than those of 0.1 viscosity contrast tests.  There are no secondary plumes and the whole system tends to reach stability after around 500 time steps.
 
+
+\paragraph{Computation of the van Keken Problem with the Volume-of-Fluid Interface Tracking Method}
```

Comment:
I think that you should just make this a `\subsubsection` like the other benchmarks/cookbooks. You should also put the note at the top that documents who contributed the section (i.e., *you*).

Reply:
@bangerth This was a collaboration between @class4kayaker and me.  He wrote the first draft and made the computations.  I wrote the final draft.

Reply:
@egpuckett -- just add something like that as the first sentence of the subsubsection, like is done for most of the other subsubsections.